openweathermap-ios/openweathermap-ios/Maps/Model/Annotation.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Annotation.swift
+//  openweathermap-ios
+//
+
+import MapKit
+
+class Annotation: NSObject, MKAnnotation {
+    var coordinate: CLLocationCoordinate2D
+    var title: String?
+    var subtitle: String?
```

Comment:
Do you need to keep it variable?

Reply:
Yes. Would you have a suggestion?

Reply:
You are receiving it by Constructor. Are you changing it later? If you don't, let all them immutable.